pkg/pluginmanager/manager.go

```diff
@@ -767,14 +760,13 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 		}()
 
 		// Initialize the spinner
-		sow, spinnerErr = component.NewOutputWriterSpinnerWithSpinnerOptions(os.Stderr, component.TableOutputType, installingMsg, true, spinnerOptions)
-		if spinnerErr != nil {
-			log.V(6).Infof("Unable to initialize spinner: %v", spinnerErr.Error())
-			log.Info(installingMsg)
-		}
-		if sow != nil {
-			defer sow.StopSpinner()
-		}
+		spinner, _ = component.NewOutputWriterSpinner(component.WithOutputStream(os.Stderr),
```

Comment:
If the API returns an error then we should handle it. That way if in a future version of the runtime we do return an actual error, this code will be ready.

Reply:
We updated the API in the PluginRuntime and now it does not return an error. Updated PR accordingly.